department-of-veterans-affairs/next-build
# 8629 qa group section

## Description
issues:
 [#8629](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8629)
[#17322](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17322)

Changes
Adds q_a_section and q_a_group queries/tests/types
adds q_a_section component with tests
adds qaCollapsible panel component with tests
modifies existing qaParagraph component and types slightly

## Testing done
Local Visual
verified components render on R&S pages
## Screenshots
<img width="1077" alt="Screen Shot 2024-03-07 at 10 33 20 AM" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/51b5fd08-0976-453b-8bda-b77656d4f32e">
<img width="1077" alt="image" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/61baba8c-597f-42e7-aec5-7c4ea2cb65b7">


## QA steps
Verify components render as expected in storybook
Verify QA components render on R&S pages, (ie: http://localhost:3000/resources/creating-an-account-for-vagov/)

```[tasklist]
```

## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
